README.Rmd

```diff
@@ -20,6 +20,23 @@ knitr::opts_chunk$set(
 The `mmrm` package implements mixed models for repeated measures (MMRM) in R
 based on Template Model Builder (TMB).
```

Comment:
```suggestion
The `mmrm` package implements mixed models for repeated measures (MMRM) in R.
```

Reply:
then it is not confusing anybody, and we can elaborate on this more in the algorithm vignette